docs/INSTALL.md

```diff
@@ -55,7 +55,7 @@ Then build and install a local copy of libsecp256k1 for python-bitcointx:
 
     mkdir -p deps
     cd deps
-    git clone git://github.com/bitcoin-core/secp256k1
+    git clone git@github.com:bitcoin-core/secp256k1
     cd secp256k1
     git checkout 0d9540b13ffcd7cd44cc361b8744b93d88aa76ba
```

Comment:
This also wrong. Should be `490022745164b56439688b0fc04f9bd43578e5c3` here and elsewhere in file, to match `install.sh`, see #1098. Could you change also that?